features/step_definitions/scheduler.rb

```diff
@@ -0,0 +1,19 @@
+Given /^the #{QUOTED} scheduler CRD is restored after scenario$/ do |name|
+  ensure_admin_tagged
+  org_scheduler = {}
+  @result = admin.cli_exec(:get, resource: 'scheduler', resource_name: name, o: 'yaml')
+  if @result[:success]
+    org_scheduler['spec'] = @result[:parsed]['spec']
+    logger.info "scheduler restore tear_down registered:\n#{org_scheduler}"
+  else
+    raise "Could not get scheduler: #{name}"
+  end
+  patch_json = org_scheduler.to_json
+  _admin = admin
+  teardown_add {
+    opts = {resource: 'scheduler', resource_name: name, p: patch_json, type: 'merge' }
+    @result = _admin.cli_exec(:patch, **opts)
+    raise "Cannot restore scheduler: #{name}" unless @result[:success]
+    sleep 60
+  }
+end
```

Comment:
Seems what this step does is common for many other CRs, could check if others defined. If not, seems better to make it common, not only to scheduler CR

Reply:
@xingxingxia features/step_definitions/oauth.rb has defined the oauth CRD.

Reply:
Just a suggestion to avoid same code multiple times. Fine to keep PR as is.